### ISSUE TITLE
import for HTTPBuilder or UriBuilder failing

### ISSUE THREAD

**Issue by prem911** [timroes/groovyplayground#14]
import groovyx.net.http.HTTPBuilder
import groovyx.net.http.UriBuilder

The above is failing and was recently working.
What could be workarounds for this?

**Comment by timroes** [comment]
Do you mean, using them inside a Groovy script you put into the Playground has been working and now isn't?

If so, I haven't updated anything around the playground for a long time (especially not the Groovy version), so the only thing I could imagine that would break that is if Google AppEngine (where this is currently hosted), blacklisted the usage of some specific Groovy APIs inside Groovy projects.

**Comment by prem911** [comment]
> Do you mean, using them inside a Groovy script you put into the Playground has been working and now isn't?
> 
> If so, I haven't updated anything around the playground for a long time (especially not the Groovy version), so the only thing I could imagine that would break that is if Google AppEngine (where this is currently hosted), blacklisted the usage of some specific Groovy APIs inside Groovy projects.

Thanks, this is quite possible. However, when I try running this in jdoodle's groovy playground, I hit the same issue. So, wondering if this is a global outage for our favourite groovy :(

**Comment by timroes** [comment]
Yeah if it suddenly started failing also in other groovy playgrounds out there, this really looks more like some issue in the infrastructure used by those. I unfortunately can't see directly if jdoodle is using GCP too, since they are running behind cloudflare, so it could be that they are running on GCP too, and it's really a GCP issue.
